run/markdown-preview/renderer/requirements.txt

```
@@ -1,5 +1,5 @@
 Flask==3.0.3
 gunicorn==22.0.0
 Markdown==3.7
-bleach==6.1.0
+bleach==6.2.0
```

6.2.0 drops Python 3.8 support
```suggestion
bleach==6.2.0; python_version > "3.9"
bleach==6.1.0; python_version <= "3.8"
```